<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Browser</class>
 <widget class="QWidget" name="Browser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>887</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MoonPlayer - Browser</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,1,0">
   <item row="0" column="0" colspan="3">
    <widget class="QLineEdit" name="urlEdit"/>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="goButton">
     <property name="text">
      <string>Go</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Favorite</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" rowspan="3" colspan="2">
    <widget class="QWebEngineView" name="webEngineView">
     <property name="url">
      <url>
       <string>about:blank</string>
      </url>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="addButton">
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="removeButton">
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Remove</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">QtWebEngineWidgets/QWebEngineView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
